src/wdio/services/ServeStaticService.js

```diff
@@ -25,12 +80,12 @@ export default class ServeStaticService {
     }
 
     const serveOptions = {
-      ...site && { site, disk: true },
+      ...site && { site },
```

Comment:
We should drop site and config from these options and pass the correct one to the service that will use them

Reply:
We're effectively doing that. If site is added to serve options we use serve static. If it's not added, we use webpack dev server.